ginivision/src/main/java/net/gini/android/vision/internal/pdf/RendererPreLollipop.java

```diff
@@ -11,12 +13,18 @@
 
     @Override
     public void toBitmap(@NonNull final Size targetSize,
-            @NonNull final Callback callback) {
-        callback.onBitmapReady(null);
+            @NonNull final AsyncCallback<Bitmap> asyncCallback) {
+        asyncCallback.onSuccess(null);
+    }
+
+    @Override
+    public void getPageCount(@NonNull final AsyncCallback<Integer> asyncCallback) {
+        asyncCallback.onSuccess(0);
     }
 
     @Override
     public int getPageCount() {
```

Comment:
Does this method make sense anymore? (Duplicated)

Reply:
It's the syncrhonous version of the other one. We need it when we do the pdf page count check in the [FileImportValidator](https://github.com/gini/gini-vision-lib-android/blob/87c3213d40b7dd6eecb0f341cfed5de58b7350f8/ginivision/src/main/java/net/gini/android/vision/internal/util/FileImportValidator.java#L101).